docs/source/examples.md

```diff
@@ -22,3 +20,6 @@
 
 # Acceleration on NPU
 [Inception model optimization on Qualcomm NPU](https://github.com/microsoft/Olive/tree/main/examples/snpe/inception_snpe_qualcomm_npu)
+
+# Acceleration on DPU
+[ResNet optimization with Vitis-AI Post Training Quantization](https://github.com/microsoft/Olive/tree/main/examples/resnet#resnet-optimization-with-vitis-ai-ptq-on-dpu)
```

Comment:
Suggest adding the HW info in the title "ResNet optimization with Vitis-AI Post Training Quantization for AMD DPU"